EIPS/eip-7748.md

```diff
@@ -246,7 +246,24 @@ We decided to not do this to reduce the algorithm complexity. Considering the cu
 
 ### Expected time for the conversion to finish
 
-TODO: We have an estimation, but it might be worth recalculating it closer to the proposed fork having a more up to date state size estimate.
+Based on current Ethereum mainnet state metrics (as of March 2024):
+- Total number of accounts: ~130M
```

Comment:
Some days ago I gathered some metrics, and this number is way too outdated.
For example, the total number of accounts today is around 275m accounts.

Also, the total contract code mentioned below is off by more than 20x.